### DISCUSSION TITLE
Multiple plots in the same cell in Observable

### DISCUSSION THREAD

**Discussion by ltostes** [observablehq/plot#482] [Q&A] [answered]
Hello all!

First, thank you all for the amazing work on ObservableHQ and Plot. They are all a breeze to use and learn.

Do you guys have any suggestion on how to combine multiple Plots in the same cell inside Observable? I realize I can insert them next to each other as they return the svg tags, but I thought I might be missing an easier way to do it.

My intention would be for instance to have a single cell with two horizontally concatenated plots that have their own X axis but share the same Y axis. I did look around the facetting part but I couldn't figure out how to use it but have separate X axis for instance.

Is there any easy way to achieve that? And if not, what should I dive into to get it?

Thanks a lot.

Best

**Comment by Fil** [comment, marked as answer]
Faceting shares all the scales between the facets. It's possible (but not easy) to trick it into using different scales, like we do in https://observablehq.com/@fil/plot-splom and https://observablehq.com/@fil/plot-parcoords ; other than that, I think you'd have to assemble subplots with [htl](https://observablehq.com/@observablehq/htl).

**Reply by ltostes** [reply]
Thanks a lot, @Fil ! 
I'll dive a bit deeper into nice ways to use htl for it
